packages/notifier/src/types-ambient.js

```diff
@@ -50,21 +50,47 @@
 
 /**
  * @template T
- * @typedef {ERef<PublicationRecord<T>>} PublicationList
- * Will be shared between machines, so it must be safe to expose. But other
- * software should consider it opaque, not depending on its internal structure.
+ * @typedef {Promise<PublicationRecord<T>>} PublicationList
+ * Will be shared between machines, so it must be safe to expose. But software
+ * outside the current package should consider it opaque, not depending on its
+ * internal structure.
  */
 
 /**
  * @template T
- * @typedef {object} Subscriber
+ * @typedef {object} EachTopic
  * @property {(publishCount?: bigint) => PublicationList<T>} subscribeAfter
  * Internally used to get the "current" SharableSubscriptionInternals
  * in order to make a subscription iterator that starts there.
  * The answer is "current" in that it was accurate at some moment between
  * when you asked and when you got the answer.
```

Comment:
The mention of SharableSubscriptionInternals here feels inappropriate, and I don't think `subscribeAfter` is limited to internal use.
```suggestion
 * @property {(publishCount?: bigint) => PublicationList<T>} subscribeAfter
 * Returns a promise for a "current" PublicationRecord that follows the provided publishCount
 * and references its successor.
 * Used to make forward-lossless ("each") iterators.
```